.github/workflows/continuous-build.yml

```diff
@@ -185,35 +186,50 @@ jobs:
 
       # workaround for handling input value for auto-triggered workflows
       # https://dev.to/mrmike/github-action-handling-input-default-value-5f2g
-      - name: Set AWS region value
+      - name: Set AWS region and partition values
         id: setRegion
         run: |
           USER_INPUT=${{ github.event.inputs.region }}
           # Use user input region or fall back to "us-west-2"
           REGION=${USER_INPUT:-"us-west-2"}
           echo "::set-output name=region::$REGION"
+          echo "::set-output name=partition::aws"
+
+          if [[ ${{ matrix.testcases }} == "linux_zip_cn" ]]; then
+            echo "::set-output name=region::cn-north-1"
+            echo "::set-output name=partition::aws-cn"
+          fi
```

Comment:
Won't this print out twice for CN regions?

Seems like 195 & 196 should be in an else if block below?

Reply:
The `region` and `partition` values will be overwritten to CN region in this case. But you provide a good point of improvement here. :)